### PR TITLE
Provide a more readable indentation.

### PR DESCRIPTION
**Pull request by bangerth**
I swear, and I hope you trust me, that I'm not trying to be passive aggressive
here -- I just had to look up a function for #6712 and I think the non-formatted
version of the formula is easier to read.

### CODE REVIEW

File: include/deal.II/matrix_free/cuda_matrix_free.h [hunk 1]
```diff
@@ -307,17 +307,20 @@ namespace CUDAWrappers
...
-             dim == 3 ? (fe_degree == 1 ? 8 : fe_degree == 2 ? 2 : 1) : 1;
+    /* clang-format off */
+    return dim==2 ? (fe_degree==1 ? 32 :
+                     fe_degree==2 ? 8 :
+                     fe_degree==3 ? 4 :
+                     fe_degree==4 ? 4 :
+                     1) :
+           dim==3 ? (fe_degree==1 ? 8 :
+                     fe_degree==2 ? 2 :
+                     1) : 1;
```
Comment (tamiko) [inline include/deal.II/matrix_free/cuda_matrix_free.h]: You have to `/* clang-format on */` again.

Comment (tamiko) [reply include/deal.II/matrix_free/cuda_matrix_free.h]: Alternatively,
```
return dim == 2 ? (fe_degree == 1 ? 32 :      //
                     fe_degree == 2 ? 8 :     //
                       fe_degree == 3 ? 4 :   //
                         fe_degree == 4 ? 4 : //
                           1) :               //
         dim == 3 ? (fe_degree == 1 ? 8 :     //
                       fe_degree == 2 ? 2 :   //
                         1) :                 //
           1;                                 //
```
(whatever looks more appealing :wink:)

Comment (tamiko) [reply include/deal.II/matrix_free/cuda_matrix_free.h]: The trailing `//` variant has one minor advantage, namely that the code block is still indented (we merely force a line break.)

Comment (tamiko) [reply include/deal.II/matrix_free/cuda_matrix_free.h]: But I am fine with either version. Change, or merge as you want :-)

Comment (bangerth) [reply include/deal.II/matrix_free/cuda_matrix_free.h]: The hell I'm going to do :-) Given how contentious this whole thing, I'm definitely not going to merge my own changes ;-)

Someone please tell me which way to go.

Comment (tamiko) [reply include/deal.II/matrix_free/cuda_matrix_free.h]: No one feels strongly about manually fixing up indentation :wink:

**Pull request merged**